canonical-web-and-design/ubuntu.com
# Update /legal/terms-and-policies/contact-us form with new form assembly code

## Done

- This form was 500'ing... so I updated with some UGLY form assembly code.
- Don't look to hard at it

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/legal/terms-and-policies/contact-us
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the form submits
